src/components/message/index.ts

```diff
@@ -0,0 +1,23 @@
+import tpl from './tpl.hbs';
+import Block from '../../service/Block';
+
+interface ITempMassage {
+    chatTitle: string;
+    href: string;
+    myMassage?: string;
+    chatMassage: string;
+    time: string;
```

Comment:
Почему бы не использовать тут тип Date? С ним удобнее будет работать, чем со строкой
```suggestion
    time: Date;
```

Reply:
А чтобы отформатировать дату для вывода на экран, можно написать отдельную функцию с такой сигнатурой 
```(time: Date) => string```